app/(dashboard)/mailboxes/[mailbox_slug]/[category]/tabBar.tsx

```
@@ -21,8 +21,16 @@ const tabsSchema = z.object({
 
 type Tab = z.infer<typeof tabsSchema>["tabs"][number];
 
+const generateUUID = () => {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
```

ESLint

[error] Replace `'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g,·function` with `"xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g,·function·`

(prettier/prettier)